app/services/idv/actions/verify_document_status_action.rb

```diff
@@ -17,6 +17,15 @@ def process_async_state(current_async_state)
 
         if current_async_state.done?
           process_result(current_async_state.result)
+          @flow.analytics.track_event(
+            Analytics::IDV_DOC_AUTH_SUBMITTED_IMAGE_UPLOAD_VENDOR,
+            current_async_state.result.to_h.merge(
+              state: current_async_state.pii[:state],
+              async: true,
```

Comment:
all the examples in test cases seem to have `async: false`, is it because the assertions for async are looser? Or are we missing coverage?

Reply:
I just added a logging spec for the async side, but yes there appears to be a lack of unit coverage for the async actions.